playground/frontend/integration_test/standalone_example_selector_test.dart

```diff
@@ -0,0 +1,115 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+import 'package:playground/modules/examples/components/example_list/example_item_actions.dart';
+import 'package:playground/modules/examples/components/filter/tag_bubble.dart';
+import 'package:playground/modules/examples/components/filter/type_bubble.dart';
+import 'package:playground/modules/examples/components/search_field/search_field.dart';
+import 'package:playground/modules/examples/examples_dropdown_content.dart';
+import 'package:playground/pages/standalone_playground/notifiers/example_selector_state.dart';
+import 'package:playground_components_dev/playground_components_dev.dart';
+import 'package:provider/provider.dart';
+
+import 'common/common.dart';
+import 'common/common_finders.dart';
+
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  testWidgets('Example selector test', (WidgetTester wt) async {
+    await init(wt);
+    await _checkFilteringExamplesByTags(wt);
+    await _checkFilteringExamplesBySearchString(wt);
```

Comment:
Check the actual examples before filtering and after filtering.

Reply:
done